cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -365,6 +387,19 @@ private void performUpgrade() throws Exception {
     applicationLifecycleService.upgrade(true);
   }
 
+  private void performHBaseUpgrade() throws Exception {
+
+    System.setProperty(AbstractHBaseDataSetAdmin.SYSTEM_PROPERTY_FORCE_HBASE_UPGRADE, Boolean.TRUE.toString());
```

Comment:
we can't do that because performHBaseUpgrade sets a system property. But we can factor it out into a method. I did not think that was necessary because it is really small but will do.